src/components/profile/profile-edit.component.tsx

```diff
@@ -227,11 +230,11 @@ export const ProfileEditComponent: React.FC<ProfileEditProps> = ({ user, toggleP
                   name="publickey"
                   disabled={true}
                   fullWidth={true}
-                  defaultValue={user.id}
+                  defaultValue={encodeAddress(user.id, 42)}
```

Comment:
bor 42 itu prefix address kah? kalau iya, karena 42 ini default substrate, daripada harcode mending ga usah kasih param prefix address (42) how?

Reply:
@immhrmwn mending lu ganti atau tambah fungsinya , `encodeToDefaultAddress` disitu lu hardcode 42 nya, acem @irmannmal ?

Reply:
yups bor, sebenernya gini doang juga **encodeAddress(user.id)** == encodeAddress(user.id, 42), liat di method nya prefix ini optional, which is kalau ga di kasih prefix default nya emang 42, suggest gua si **encodeAddress(user.id)** aja

Reply:
okey, done